src/Bicep.Core/Bicep.Core.csproj

```diff
@@ -1,19 +1,30 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>netstandard2.1</TargetFramework>
+	</PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="JetBrains.Annotations" Version="2020.3.0">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
-    <PackageReference Include="Azure.Deployments.Expression" Version="1.0.168" />
-    <PackageReference Include="Azure.Deployments.Core" Version="1.0.168" />
-    <PackageReference Include="Azure.Bicep.Types" Version="0.1.95" />
-    <PackageReference Include="Azure.Bicep.Types.Az" Version="0.1.95" />
-  </ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="JetBrains.Annotations" Version="2020.3.0">
+			<PrivateAssets>all</PrivateAssets>
+		</PackageReference>
+		<PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.0-preview.3.21201.4" />
+		<PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0-preview.3.21201.4" />
+		<PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0-preview.3.21201.4" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+		<PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
+		<PackageReference Include="Azure.Deployments.Expression" Version="1.0.168" />
+		<PackageReference Include="Azure.Deployments.Core" Version="1.0.168" />
+		<PackageReference Include="Azure.Bicep.Types" Version="0.1.95" />
+		<PackageReference Include="Azure.Bicep.Types.Az" Version="0.1.95" />
+	</ItemGroup>
 
+	<ItemGroup>
+		<None Update="bicepsettings.json">
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+		</None>
+	</ItemGroup>
```

Comment:
Embedding this as an assembly resource feels like it would be more reliable rather than including as a separate file.

Reply:
done